include/git2/diff.h

```diff
@@ -130,7 +130,9 @@ typedef enum {
 	GIT_DIFF_INCLUDE_CASECHANGE = (1u << 11),
 
 	/** If the pathspec is set in the diff options, this flags means to
-	 *  apply it as an exact match instead of as an fnmatch pattern.
```

Comment:
Say:

file1.ext
file1.ext.generated

What will the diff on file1.ext do? Before it would've matched `file1.ext`.